yt_dlp/extractor/art19.py

```diff
@@ -0,0 +1,304 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import float_or_none, int_or_none, parse_iso8601, url_or_none
+from ..utils.traversal import traverse_obj
+
+
+class Art19IE(InfoExtractor):
+    _UUID_REGEX = r'[\da-f]{8}-?[\da-f]{4}-?[\da-f]{4}-?[\da-f]{4}-?[\da-f]{12}'
```

Comment:
I think a UUID without dashes is not valid. Unless you have examples of this site using invalid UUIDs, I would simplify the pattern by enforcing dashes
```suggestion
    _UUID_REGEX = r'[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12}'
```
&nbsp;
could do this to make it shorter, although arguably harder to read:
```suggestion
    _UUID_REGEX = r'[\da-f]{8}(?:-[\da-f]{4}){3}-[\da-f]{12}'
```
really this should be a util by now imo

Reply:
The site accepts both 
`https://art19.com/shows/scamfluencers/episodes/8319b776-4153-4d22-8630-631f204a03dd` (with all dashes)
and
`https://art19.com/shows/scamfluencers/episodes/8319b77641534d228630631f204a03dd` (no dashes)
but not partial dashes or dashes in different places